ironfish-cli/src/commands/service/stats.ts

```diff
@@ -188,4 +189,65 @@ export default class Stats extends IronfishCommand {
       await PromiseUtils.sleep(delay)
     }
   }
+
+  async chainReorgs(api: WebApi): Promise<void> {
+    let connected = false
+
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
+      connected = await this.sdk.client.tryConnect()
+
+      if (!connected) {
+        await PromiseUtils.sleep(1000)
+        continue
+      }
+
+      const response = this.sdk.client.event.onReorganizeChainStream()
+
+      for await (const value of response.contentStream()) {
+        const { oldHead, newHead, fork } = value
+        void api.submitTelemetry({
+          points: [
+            {
+              measurement: 'chain_reorg',
+              timestamp: new Date(),
+              fields: [
+                {
```

Comment:
Should we also just put a field in here with the length? Or do you want to calculate that in influxdb?

Reply:
I was thinking of pushing that calculation to the influx query since it can be derived from the fields here

Reply:
That sounds good to me.